files/k8s-resource/calico-all.yaml

```diff
@@ -309,6 +309,11 @@ spec:
         # priority scheduling and that its resources are reserved
         # if it ever gets evicted.
         scheduler.alpha.kubernetes.io/critical-pod: ''
+        prometheus.io/scrape: "true"
+        prometheus.io/port: "9091"
+        giantswarm.io/monitoring: "true"
```

Comment:
Note that this will need to be a label instead of an annotation, see note in the [draft doc](https://github.com/giantswarm/giantswarm/blob/245a6a193e7e0f14b87563c139e6e31b8966be8c/content/docs/architecture-specs-adrs/specs/configuration-of-targets-in-tc-prometheus.md#labels-and-annotations).